python/paddle/fluid/contrib/mixed_precision/fp16_lists.py

```diff
@@ -71,11 +71,7 @@ def _update_list(self):
 
 # The set of ops that support fp16 calculation and are considered numerically-
 # safe and performance-critical. These ops are always converted to fp16.
-white_list = {
-    'conv2d',
-    'matmul',
-    'mul',
-}
+white_list = {'conv2d', 'matmul', 'mul', 'fused_bn_add_activation'}
```

Comment:
I think `fused_bn_add_activation` should be added in the `gray_list `.

Reply:
done.